### PR TITLE
Remove office-toolbox package

### PR DESCRIPTION
**Pull request by akrantz**
Instead of using office-toolbox for sideload and unload, the ability to register, unregister, and sideload add-ins is being integrated directly into office-addin-debugging start/stop.

The package.json scripts to sideload/unload are being removed, as they are not needed.

NOTE: this will affect documentation.
https://docs.microsoft.com/en-us/office/dev/add-ins/testing/sideload-office-addin-using-sideload-command

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,19 +17,11 @@
...
     "start": "office-addin-debugging start manifest.xml",
     "start:desktop": "office-addin-debugging start manifest.xml desktop",
     "start:web": "office-addin-debugging start manifest.xml web",
     "stop": "office-addin-debugging stop manifest.xml",
     "test": "mocha -r ts-node/register test/*.ts",
-    "unload": "echo Please specify which app using 'npm run unload:app'.",
-    "unload:excel": "office-toolbox remove -m manifest.xml -a excel",
-    "unload:powerpoint": "office-toolbox remove -m manifest.xml -a powerpoint",
-    "unload:word": "office-toolbox remove -m manifest.xml -a word",
     "validate": "office-toolbox validate -m manifest.xml",
```
Comment (akrantz) [inline package.json]: Need to move "office-toolbox validate" over to "office-addin-manifest validate"...

Comment (akrantz) [reply package.json]: This is now completed.

**Comment by akrantz** [comment]
@kbrandl This change to the templates removes the "sideload" and "unload" scripts from package.json. It is no longer needed because "office-addin-debugging start" has been enhanced to be able to call the sideload directly. The docs should already discuss using "npm start", but we might have some older documentation that refers to using "npm run sideload". I referenced the page that I found when I searched, but please search through the documentation and make sure that we understand the documentation impact and let us know if you're OK with these changes.

**Comment by akrantz** [comment]
This is now working with the support now provided by the office-addin-debugging and office-addin-manifest packages.

**Comment by akrantz** [comment]
I though I had already made the change to use office-addin-manifest instead of office-toolbox to validate the manifest, but now it's really there.

**Comment by TCourtneyOwen** [comment]
I am looking at the CI loop run and notice that the taskpane is not loading as expected in the UI tests. I am going to check out this branch locally and see if the UI tests pass. If so then it must be an issue with the VMs

**Comment by TCourtneyOwen** [comment]
tests pass locally for this branch - need to figure out what's wrong with the CI loop. The tests were hanging because the taskpane wasn't loading successfully.  Could be a sign-in disabling problem, which was an issue with the 1809 agents

**Comment by TCourtneyOwen** [comment]
job now passes for your PR. Perhaps it was the loopback problem.  I deleted the whole _work/1 directory, ran the loop with your changes and this time it passed with no problems: https://office.visualstudio.com/OC/_build/results?buildId=2725469

**Comment by akrantz** [comment]
@TCourtneyOwen I went ahead and took care of it.

**Comment by TCourtneyOwen** [comment]
great, thanks!

**Comment by kbrandl** [comment]
FYI, related doc updates are in this PR: https://github.com/OfficeDev/office-js-docs-pr/pull/1252.

**Pull request merged**